tools/md5sum.py

```diff
@@ -0,0 +1,16 @@
+import io
+import hashlib
+
+
+def md5sum(path):
```

Comment:
There's a name clash here with the sub-module name, but this will go away once we move everything into a single module